### PR TITLE
README: add registry documentation link.

### PR DESCRIPTION
**Pull request by PI-Victor**
A link to the documentation describing what this is part of and how to get started would be useful.

~~The caveat is it will be out-of-date with each doc version release.~~ 

s/Promethus/Prometheus

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,9 +9,12 @@ OpenShift Image Registry
 
 ***OpenShift Image Registry*** is a tightly integrated with [OpenShift Origin](https://www.openshift.org/) application that lets you distribute Docker images.
 
+Installation and configuration instructions can be found in the
+[OpenShift documentation](https://docs.okd.io/3.10/install_config/registry/index.html).
```
[the remark this answers is not in the document]
Comment (PI-Victor) [reply README.md]: sorry, i forgot about "latest" - fixed.

**Comment by bparees** [comment]
thanks @PI-Victor !  nice to see you again :)

**Comment by PI-Victor** [comment]
thanks! likewise @bparees :)

**Pull request merged**